crates/extensions/c8y_mapper_ext/src/tests.rs

```diff
@@ -2086,27 +2086,37 @@ async fn json_custom_operation_status_multiple_operations_in_one_mqtt_message()
     );
     mqtt.send(input_message).await.expect("Send failed");
 
-    assert_received_contains_str(&mut mqtt, [("c8y/s/us", "504,111")]).await;
-    assert_received_contains_str(&mut mqtt, [("c8y/s/us", "504,222")]).await;
-    assert_received_contains_str(&mut mqtt, [("c8y/s/us", "504,333")]).await;
+    let mut messages = vec![];
+    for _ in 0..6 {
+        messages.push(mqtt.recv().await.unwrap());
+    }
+    let (mut requests, mut responses): (Vec<_>, Vec<_>) = messages
+        .iter()
+        .map(|msg| (msg.topic.name.as_str(), msg.payload.as_str().unwrap()))
+        .partition(|(_topic, payload)| payload.starts_with("504,"));
 
+    // The messages might get processed out of order, we don't care about the ordering of the messages
+    requests.sort();
```

Comment:
This probably isn't technically necessary as I imagine the requests are always sent in the order from the original c8y message, but I think the point still stands that the ordering is irrelevant.